### PR TITLE
ADEN-5663 Replay ads for every next FV

### PR DESCRIPTION
**Pull request by v3rth**
@Wikia/x-wing 
"Don't play ads between videos"
Is it only business reason or there something technical behind it?

### CODE REVIEW

File: extensions/wikia/ArticleVideo/scripts/ooyala-player.js [hunk 1]
```diff
@@ -42,6 +42,7 @@ define('ooyala-player', ['wikia.browserDetect'], function (browserDetect) {
 
 		messageBus.subscribe(window.OO.EVENTS.PLAYBACK_READY, 'ui-update', function () {
 			self.onPlaybackReady();
+			// messageBus.publish("replay", false);
```
Comment (WikiaTech) [inline extensions/wikia/ArticleVideo/scripts/ooyala-player.js]: MAJOR Remove this commented out code. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3ACommentedCode)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 2 issues
* ![MAJOR][MAJOR] 2 major

Watch the comments in this conversation to review them.

#### 1 extra issue

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [adContext.js#L78](https://github.com/Wikia/app/blob/1e65b21c6b331504f157b0c751839193e146cd71/extensions/wikia/AdEngine/js/context/adContext.js#L78): Reduce the number of conditional operators (4) used in the expression (maximum allowed 3). [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1067)

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

**Comment by WikiaTech** [comment]
SonarQube analysis reported 1 issue

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [adContext.js#L78](https://github.com/Wikia/app/blob/ba2abdcdf8576722e3ff278edcc57a2f3e4c69b1/extensions/wikia/AdEngine/js/context/adContext.js#L78): Reduce the number of conditional operators (4) used in the expression (maximum allowed 3). [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1067)

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

**Comment by WikiaTech** [comment]
SonarQube analysis reported 1 issue

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [adContext.js#L78](https://github.com/Wikia/app/blob/9ab657bf5ae9073ab627980c8c0eef3b7d22d596/extensions/wikia/AdEngine/js/context/adContext.js#L78): Reduce the number of conditional operators (4) used in the expression (maximum allowed 3). [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1067)

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

File: extensions/wikia/AdEngine/js/context/adContext.js [hunk 2]
```diff
@@ -258,6 +258,8 @@ define('ext.wikia.adEngine.adContext', [
 		context.opts.premiumAdLayoutRubiconFastlaneTagsEnabled = context.opts.premiumAdLayoutEnabled &&
 			geo.isProperGeo(instantGlobals.wgAdDriverPremiumAdLayoutRubiconFastlaneTagsCountries);
 
+		context.opts.replayAdsForFVVideos = geo.isProperGeo(instantGlobals.wgAdDriverReplayAdsForFVVideosCountries);
```
Comment (latata) [inline extensions/wikia/AdEngine/js/context/adContext.js]: FVVideos = Featured Video Videos?

**Review by latata — APPROVED** [review]

**Comment by WikiaTech** [comment]
SonarQube analysis reported 1 issue

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [adContext.js#L78](https://github.com/Wikia/app/blob/f0909d9ebaf5e10fdf0f125e94a5ad609c5b113c/extensions/wikia/AdEngine/js/context/adContext.js#L78): Reduce the number of conditional operators (4) used in the expression (maximum allowed 3). [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1067)

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

**Comment by WikiaTech** [comment]
SonarQube analysis reported 1 issue

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [adContext.js#L78](https://github.com/Wikia/app/blob/e0ea3a7120edd7b59e1e82671f9020ed7c8a5d14/extensions/wikia/AdEngine/js/context/adContext.js#L78): Reduce the number of conditional operators (4) used in the expression (maximum allowed 3). [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1067)

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

**Comment by WikiaTech** [comment]
SonarQube analysis reported 1 issue

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [adContext.js#L78](https://github.com/Wikia/app/blob/a2a248b5c53c017ef615f13b041da0ee3066d851/extensions/wikia/AdEngine/js/context/adContext.js#L78): Reduce the number of conditional operators (4) used in the expression (maximum allowed 3). [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1067)

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

**Review by nandy-andy — APPROVED** [review]

**Comment by WikiaTech** [comment]
SonarQube analysis reported 1 issue

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [adContext.js#L78](https://github.com/Wikia/app/blob/f3da6110426fca5c0bb836b056043e35a25eb0f9/extensions/wikia/AdEngine/js/context/adContext.js#L78): Reduce the number of conditional operators (4) used in the expression (maximum allowed 3). [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1067)

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

**Comment by WikiaTech** [comment]
SonarQube analysis reported 1 issue

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [adContext.js#L78](https://github.com/Wikia/app/blob/7298e65637365624f8d152d3cad7759e501a4718/extensions/wikia/AdEngine/js/context/adContext.js#L78): Reduce the number of conditional operators (4) used in the expression (maximum allowed 3). [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1067)

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

**Review by nandy-andy — APPROVED** [review]

**Review by nandy-andy — APPROVED** [review]

**Pull request merged**